gollm/ollama.go

```diff
@@ -37,7 +37,7 @@ func ollamaFactory(ctx context.Context, opts ClientOptions) (Client, error) {
 }
 
 const (
-	defaultOllamaModel = "gemma2:latest"
+	defaultOllamaModel = "gemma3:latest"
```

Comment:
How has been your experience of running it with gemma3 ?

Would be interested in running `k8s-bench` for gemma3 and share the results ?